examples/trials/mnist-nas/oneshot_mode/config_oneshot.yml

```diff
@@ -0,0 +1,21 @@
+authorName: default
+experimentName: example_mnist
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+#choice: local, remote, pai
+trainingServicePlatform: local
+#choice: true, false
+useAnnotation: true
+tuner:
+  #choice: TPE, Random, Anneal, Evolution, BatchTuner, MetisTuner
+  #SMAC (SMAC should be installed through nnictl)
+  #codeDir: ~/nni/nni/examples/tuners/random_nas_tuner
+  codeDir: ../../../tuners/random_nas_tuner
+  classFileName: random_nas_tuner.py
+  className: RandomNASTuner
+trial:
+  command: python3 mnist-oneshot.py
+  codeDir: .
+  gpuNum: 0
+  nasMode: oneshot_mode
```

Comment:
why shall we fill in `nasMode` here? Where will it be passed?

Reply:
it is passed to nnictl pre-compile, so that the pre-compile can generate code based on the nasMode.